python/activator/activator.py

```diff
@@ -348,7 +348,7 @@ def next_visit_handler() -> tuple[str, int]:
         try:
             expected_visit = parse_next_visit(flask.request)
         except ValueError as msg:
-            _log.exception()
+            _log.exception(msg)
```

Comment:
This is existing but `msg` as an exception variable is a bit non-standard.

Reply:
```suggestion
            _log.exception("Bad request: %s", msg)
```

Oh, and probably add a context message here like you have in the return.

Reply:
Indeed. I'm changing it to `e`